src/EnergyPlus/TARCOGCommon.hh

```diff
@@ -54,6 +54,11 @@
 // EnergyPlus Headers
 #include <EnergyPlus/Data/BaseData.hh>
 #include <EnergyPlus/EnergyPlus.hh>
+#include <EnergyPlus/TARCOGParams.hh>
+
+#define BITF(B) (1 << (int(B)))
```

Comment:
Is this a second definition of BITF and BITF_TEST_ANY?  If so they need to be pulled into a single location and reused.

Reply:
Done 👍🏽